habitat_baselines/common/env_utils.py

```diff
@@ -0,0 +1,141 @@
+import random
+
+import numpy as np
+
+import habitat
+from habitat import Config, SimulatorActions, make_dataset
+
+
+class NavRLEnv(habitat.RLEnv):
```

Comment:
Place `NavRLEnv` inside a separate file under `common`.

Reply:
Looks like we will need Registry for RLEnvs and call this PointGoalRLEnv. Also add meaningful comment how RLEnvs should be used.

Reply:
Ideally the registry should support nesting, something like `registry.register_env('rl.point_nav', EnvClassHere)`.